composer.json

```diff
@@ -10,13 +10,13 @@
         }
     ],
     "require": {
-        "zendframework/zend-http": "~2.2.5",
-        "zendframework/zend-stdlib": "~2.2.5",
-        "zendframework/zend-uri": "~2.2.5"
+        "php": ">=5.5.0",
+        "zendframework/zend-stdlib": "~2.2.5"
     },
     "require-dev": {
         "phpunit/PHPUnit": "3.7.*",
         "satooshi/php-coveralls": ">=0.6.0",
+        "psr/http-message": "1.0.*@dev",
```

Comment:
I suspect that once we reach 1.0, there won't be any BC for years. This problem will likely be reduced to a minimum

Envoyé de mon iPhone

> Le 6 avr. 2015 à 19:08, Daniel Gimenes <notifications@github.com> a écrit :
> 
> In composer.json:
> 
> >      },
> >      "require-dev": {
> >          "phpunit/PHPUnit": "3.7.*",
> >          "satooshi/php-coveralls": ">=0.6.0",
> > +        "psr/http-message": "1.0.*@dev",
> Take a exemple:
> 
> MessageInterface from psr/http-message:0.1 specifies method addHeader($name, $value).
> MessageInterface from psr/http-message:0.2 specifies method withHeader($name, $value).
> 
> If there's no version constraint in composer.json, it will break in runtime if you call a method from 0.1 and I'm implementing 0.2.
> 
> —
> Reply to this email directly or view it on GitHub.
>